spark-rapids/spark-rapids.sh

```diff
@@ -8,89 +8,39 @@ function get_metadata_attribute() {
   /usr/share/google/get_metadata_value "attributes/${attribute_name}" || echo -n "${default_value}"
```

Comment:
ideally this script has a license, copyright and description at the top like the mig.sh script does

Reply:
Thanks for pointing it out. I have added the license and a description for the script.